### ISSUE TITLE
Discount text is vocalized as a list but there is only one item

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13588]
**Describe the bug**

- Discount text is announced as a list with only one item, it's confusing to the user as usually more than 1 item is expected in a list and screen reader instructions on how to move are provided automatically by the screen reader.
- Do we have more several items scenarios?

**Tell us the version of Spartacus**
- Library version: 4.0.0.

**To Reproduce**

1. Open any Screen Reader (VO, NVDA, JAWS).
2. Browse page on "read mode" (aux keys + arrow keys)
3. Reach the discount text.
4. Listen to the description as a being a list.

**Expected behavior**
- Alternate between standard <p> for one discount and list <ul> if more items need to be added to that area.

**Screenshots**
![Screen Shot 2021-08-31 at 4 34 47 PM](https://user-images.githubusercontent.com/15113219/131573175-80cdfa6b-08b7-412d-9dae-612eec8f7ca2.png)

**Comment by saad-mo** [comment]
need to change markup and render a conditional list for multiple promos or just a `p` for single promo

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.

Tested that one promotion is a paragraph.

<img width="1433" alt="Screen Shot 2021-12-08 at 12 18 56 PM" src="https://user-images.githubusercontent.com/45409515/145270112-b8991aeb-c75f-47c5-be6b-2f589bf4d828.png">

And more than one promotion is a list.
<img width="1436" alt="Screen Shot 2021-12-08 at 12 03 59 PM" src="https://user-images.githubusercontent.com/45409515/145270512-987f085a-85a4-446c-b0fa-6928aa313299.png">


But when the focus is on each promotion, it says "list 1 item". it's like each promo itself is a list? Shouldn't it say list of 2 items...?
![promoList1](https://user-images.githubusercontent.com/45409515/145270084-4d558c17-7320-4531-bc64-e7a961299d6c.png)

**Comment by KateChuen** [comment]
Re-opening the ticket after confirming with Miguel that it should be e.g. "list with 3 items":
<div>single promo </div> or,
<ul> <li>promo 1</li> <li>promo 2</li> <li>promo 3</li> </ul>

**Comment by KateChuen** [comment]
Retested, all good now. 
One promo = 1 paragraph
<img width="1437" alt="Screen Shot 2022-01-11 at 3 06 45 PM" src="https://user-images.githubusercontent.com/45409515/149017752-8a1b5c7b-e197-4990-b975-94060671b29c.png">

More than 1 promo = list
<img width="1440" alt="Screen Shot 2022-01-11 at 3 15 12 PM" src="https://user-images.githubusercontent.com/45409515/149017750-f65d4dc0-b9c5-42aa-97ca-77a3ed569d96.png">



In read mode, it says "list of x items", then reads through the list of promotions. 
![promoList4](https://user-images.githubusercontent.com/45409515/149017738-82f5ad77-1343-4d3d-b539-071a59ebde4c.png)

![promoList3](https://user-images.githubusercontent.com/45409515/149017735-16fca7d9-520a-4551-ac9f-33212cf76cc7.png)

![promoList5](https://user-images.githubusercontent.com/45409515/149017739-6e28dbcb-517b-439f-938b-26ed03550d27.png)
